README.md

```diff
@@ -12,84 +12,103 @@ The latest versions and installation options are available at the [InSpec](http:
 config_db_users: []
 
-# description: List of authorized users of the admn database (e.g., ['[ { "role" : "userAdminAnyDatabase", "db" : "admin" } ] }'])
+# Description: List of authorized users of the admn database (e.g., ['[ { "role" : "userAdminAnyDatabase", "db" : "admin" } ] }'])
 myUserAdmin_allowed_role: []
 
-# description: List of authorized users of the admn database (e.g., ['[ { "role" : "userAdminAnyDatabase", "db" : "admin" } ] }'])
+# Description: List of authorized users of the admn database (e.g., ['[ { "role" : "userAdminAnyDatabase", "db" : "admin" } ] }'])
 mongoadmin_allowed_role: []
 
-# description: List of authorized users of the admn database (e.g., ['[ { "role" : "readWrite", "db" : "config" }, { "role" : "clusterAdmin", "db" : "admin" } ] }'])
+# Description: List of authorized users of the admn database (e.g., ['[ { "role" : "readWrite", "db" : "config" }, { "role" : "clusterAdmin", "db" : "admin" } ] }'])
 appAdmin_allowed_role: []
 
-# description: List of authorized users of the admn database (e.g., ['[ { "role" : "clusterAdmin", "db" : "admin" }, { "role" : "readAnyDatabase", "db" : "admin" }, { "role" : "readWrite", "db" : "config" } ] }'])
+# Description: List of authorized users of the admn database (e.g., ['[ { "role" : "clusterAdmin", "db" : "admin" }, { "role" : "readAnyDatabase", "db" : "admin" }, { "role" : "readWrite", "db" : "config" } ] }'])
 accountAdmin01_allowed_role: []
 ```
 
-## Running This Overlay
+## Running This Overlay Directly from Github
+
+```
+# How to run
+inspec exec https://github.com/CMSgov/cms-ars-3.1-moderate-mongodb-enterprise-advanced-3-stig-overlay/archive/master.tar.gz --input-file=<path_to_your_inputs_file/name_of_your_inputs_file.yml> --reporter=cli json:<path_to_your_output_file/name_of_your_output_file.json>
+```
+
+### Different Run Options
+
+  [Full exec options](https://docs.chef.io/inspec/cli/#options-3)
+
+## Running This Overlay from a local Archive copy 
+
+If your runner is not always expected to have direct access to GitHub, use the following steps to create an archive bundle of this overlay and all of its dependent tests:
+
+(Git is required to clone the InSpec profile using the instructions below. Git can be downloaded from the [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) site.)
+
 When the __"runner"__ host uses this profile overlay for the first time, follow these steps: 
 
 ```
 mkdir profiles
 cd profiles
-git clone https://github.com/mitre/mongodb-enterprise-advanced-3-stig-baseline.git
 git clone https://github.com/CMSgov/cms-ars-3.1-moderate-mongodb-enterprise-advanced-3-stig-overlay.git
-cd cms-ars-3.1-moderate-mongodb-enterprise-advanced-3-stig-overlay
-bundle install
-cd ..
-inspec exec cms-ars-3.1-moderate-mongodb-enterprise-advanced-3-stig-overlay --attrs=<path_to_your_attributes_file/name_of_your_attributes_file.yml> --reporter=cli json:<path_to_your_output_file/name_of_your_output_file.json>
```

Comment:
@ejaronne The original command did not have the `--target` flag, but based on the next section - it may be needed so I've added it. Please lmk if I need to change anything

Reply:
@ejaronne approved